README.md

```diff
@@ -891,8 +901,8 @@ console.log(trade);
   minAmountConvertQuote: '0.014400465273974444',
   maximumSent: null,
   expectedConvertQuote: '0.014730394044348867',
-  liquidityProviderFee: '0.030000000000000000',
-  liquidityProviderFeePercent: 0.003,
+  liquidityProviderFee: ['0.030000000000000000','0.050000000000000000','0.030000000000000000'],
+  liquidityProviderFeePercent: [0.003, 0.005, 0.003],
```

Comment:
from my memory the liquidity provider fee percent on this level is just 1 percent and if you got many routes you can go then it many objects in the array